src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -17,7 +18,27 @@ const ParentBookingCard = props => {
     location,
     instructor_name,
     size,
+    course_id,
   } = props.booking;
+
+  const token = localStorage.getItem('okta-token-storage');
+  const parsedToken = JSON.parse(token);
+
+  const handleClick = e => {
+    axios
+      .post(
+        `http://localhost:8080/children/1/enrollments`,
+        { child_id: 1, class_id: course_id, completed: true },
+        {
+          headers: {
+            Authorization: 'Bearer ' + parsedToken.idToken.value,
+          },
+        }
+      )
+      .then(res => console.log(res))
+      .catch(err => console.log(err));
```

Comment:
Should throw an error message instead of  console.log the error.

Reply:
I agree. Correct me if I'm wrong... should it be `console.log(err.message)` ?

Reply:
Mark is correct,
Maybe something like  with your own input for string ticks I put 
 .catch(err => {(‘ message : err.message’)};